src/components/interfaces/HMI_API.xml

```diff
@@ -1182,6 +1182,13 @@
   </element>
 </enum>
 
+<enum name="DeliveryMode">
```

Comment:
@anosach-luxoft  As I know if we change HMI API we should increase it's version and update date in HEAD.